AutocompleteClient/FW/Logic/Request/CIOTrackConversionData.swift

```diff
@@ -18,23 +18,59 @@ struct CIOTrackConversionData: CIORequestData {
     let customerID: String
     var sectionName: String?
     let revenue: Double?
-
+    let conversionType: String?
+    
     func url(with baseURL: String) -> String {
-        return String(format: Constants.TrackConversion.format, baseURL, self.searchTerm.addingPercentEncoding(withAllowedCharacters: .urlPathAllowed)!)
+        return String(format: Constants.TrackConversion.format, baseURL)
     }
 
-    init(searchTerm: String, itemName: String, customerID: String, sectionName: String? = nil, revenue: Double? = nil) {
+    init(searchTerm: String, itemName: String, customerID: String, sectionName: String? = nil, revenue: Double? = nil, conversionType: String? = nil) {
         self.searchTerm = searchTerm
         self.itemName = itemName
```

Comment:
keep item_name in request url, remove everything else to match other v2 endpoints

Reply:
will keep name and id in the request url for integration metrics